## Description: add some description
## Origin/Author: Shawn Landden <shawnlandden@gmail.com> and
## Daniel Gimpelevich <daniel@gimpelevich.san-francisco.ca.us>
## Bug: https://bugs.launchpad.net/ubuntu/+source/xchat/+bug/933822
Index: xchat-2.8.8/xchat.desktop
===================================================================
--- xchat-2.8.8.orig/xchat.desktop	2010-05-30 08:08:48.000000000 +0200
+++ xchat-2.8.8/xchat.desktop	2012-07-19 16:51:31.250553039 +0200
@@ -15,9 +15,10 @@
 Comment[ro]=Client de IRC
 Comment[zh_TW]=X-Chat 聊天程式
 Comment=Chat with other people using Internet Relay Chat
-Exec=xchat
+Exec=sh -c "xchat --existing --url %U || exec xchat"
 Icon=xchat
 Terminal=false
 Type=Application
 Categories=Network;
 StartupNotify=true
+MimeType=x-scheme-handler/irc;x-scheme-handler/ircs;
